From 1438a25e381f9c6bda5b3bda02ce3c68c63c9f99 Mon Sep 17 00:00:00 2001
From: "radhika.s" <radhika.s@lge.com>
Date: Tue, 26 Mar 2019 20:22:03 +0530
Subject: [PATCH 2/2] Enable NTFS file system support

:Release Notes:
Enable NTFS file system support in OSE

:Detailed Notes:
PDM requires NTFS file system to mount and do NTFS file system
operations.

:Testing Performed:
Tested and able to mount NTFS USB storage successfuly

:QA Notes:
NA

:Issues Addressed:
[PLAT-76358] Unable to mount NTFS file system
---
 arch/arm/configs/bcm2709_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index 90e267a..5e08a45 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -1280,7 +1280,7 @@ CONFIG_UDF_FS=m
 CONFIG_MSDOS_FS=y
 CONFIG_VFAT_FS=y
 CONFIG_FAT_DEFAULT_IOCHARSET="ascii"
-CONFIG_NTFS_FS=m
+CONFIG_NTFS_FS=y
 CONFIG_NTFS_RW=y
 CONFIG_TMPFS=y
 CONFIG_TMPFS_POSIX_ACL=y
-- 
2.7.4

